packages/turf-rhumb-bearing/index.js

```diff
@@ -0,0 +1,60 @@
+// https://en.wikipedia.org/wiki/Rhumb_line
+// http://www.movable-type.co.uk/scripts/latlong.html#rhumblines
+var getCoords = require('@turf/invariant').getCoords;
```

Comment:
Since you're only dealing with points you can use [`getCoord`](https://github.com/Turfjs/turf/tree/rhumb/packages/turf-invariant#getcoord) which has some extra point validation built-in.